### PR TITLE
Show a tooltip with link when clicking Farm icon

### PR DESCRIPTION
**Pull request by hungdoansy**
| Before | After |
| --- | --- |
| Clicking the Farm icon will redirect users to the Farm page immediately | Show a tooltip with a link for users taking actions on their own |
| - | image |

### CODE REVIEW

**Review by namgold — APPROVED** [review]

File: src/components/PositionCard/index.tsx [hunk 1]
```diff
@@ -111,6 +112,24 @@ const WarningMessage = styled(Text)`
...
 `
 
+const MoneyBagIcon = () => {
+  const theme = useTheme()
+  return (
+    <Flex
+      width={24}
+      height={24}
+      justifyContent="center"
+      alignItems={'center'}
```
Comment (nguyenhuudungz) [inline src/components/PositionCard/index.tsx:122]: ""

File: src/components/ProAmm/ProAmmPoolInfo.tsx [hunk 2]
```diff
@@ -14,9 +15,29 @@ import { MouseoverTooltip } from 'components/Tooltip'
 import { ELASTIC_BASE_FEE_UNIT } from 'constants/index'
 import useProAmmPoolInfo from 'hooks/useProAmmPoolInfo'
 import useTheme from 'hooks/useTheme'
+import { MEDIA_WIDTHS } from 'theme'
 import { shortenAddress } from 'utils'
 import { unwrappedToken } from 'utils/wrappedCurrency'
 
+const MoneyBagIcon = () => {
```
Comment (nguyenhoaidanh) [inline src/components/ProAmm/ProAmmPoolInfo.tsx:22]: can we reuse MoneyBagIcon?  they are the same.

Comment (nguyenhuudungz) [inline src/components/ProAmm/ProAmmPoolInfo.tsx:22]: Duplicate components?

**Review by nguyenhuudungz — CHANGES_REQUESTED** [review]

File: src/components/ProAmm/ProAmmPoolInfo.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/components/ProAmm/ProAmmPoolInfo.tsx:22]: Fixed

Comment (hungdoansy) [reply src/components/ProAmm/ProAmmPoolInfo.tsx:22]: Fixed

**Review by nguyenhuudungz — APPROVED** [review]

**Review by nguyenhoaidanh — APPROVED** [review]

**Pull request merged**